src/mapping/contentful/two-grid.tsx

```diff
@@ -10,7 +10,7 @@ export const TwoGridComponent = (props: DefaultQueryParameters) => {
     const componentData: ComponentTwoGridFieldsFragment | null | undefined = data.data?.twoGrid;
 
     const expertAdvicesProps:any = componentData?.componentLeft;
-    expertAdvicesProps.advices = [];
+    expertAdvicesProps != null ? expertAdvicesProps.advices = [] : null;
     expertAdvicesProps?.advicesCollection?.items.forEach((item) => {
```

Comment:
Can't you just use a .map to return a result array instead of starting to set the array and pushing to it inside the forEach? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map if it is not exactly the behaviour you're looking for, there might be other, similar functions that might.

Reply:
This is taken care in another PR, the 2grid becomes dynamic which becomes dynamic